src/xenia/base/filesystem_posix.cc

```diff
@@ -240,7 +247,30 @@ std::vector<FileInfo> ListFiles(const std::filesystem::path& path) {
     result.push_back(info);
   }
   closedir(dir);
-  return result;
+  return std::move(result);
+}
+
+std::vector<FileInfo> ListDirectories(const std::filesystem::path& path) {
```

Comment:
This part (``ListDirectories`` and ``FilterByName``) looks like might be directly in filesystem.cc. That way there will be only one implementation

Reply:
Done